packages/serializers/html-serializer/src/serializer/serializeHTMLFromNodes.ts

```diff
@@ -25,26 +25,36 @@ const stripSlateDataAttributes = (rawHtml: string): string =>
  */
 const stripClassNames = (
   html: string,
-  { preserveClassNames = ['slate-'] }: { preserveClassNames?: string[] }
+  { preserveClassNames = ['slate-'] }: { preserveClassNames?: string[] } = {}
```

Comment:
NOTE: this default to empty object was added unrelated to the fix, but if you were only to call `stripClassNames` with `html` it would fail previously